frontend/packages/eslint-plugin-console/lib/config/rules/typescript.js

```diff
@@ -162,7 +168,7 @@ module.exports = {
 
   // Disallow the use of variables before they are defined
   'no-use-before-define': 'off',
-  '@typescript-eslint/no-use-before-define': 'error',
+  '@typescript-eslint/no-use-before-define': 'off',
```

Comment:
Why change this?

Reply:
the linter is smarter now it seems, a lot of cases where functions where used before defined.

Reply:
We can disable just functions if we need to:

https://github.com/typescript-eslint/typescript-eslint/blob/master/packages/eslint-plugin/docs/rules/no-use-before-define.md#other-options

https://eslint.org/docs/rules/no-use-before-define#functions